package.json

```diff
@@ -115,11 +119,11 @@
   },
   "bundlesize": [
     {
-      "path": "./dist/vue-instantsearch.js",
+      "path": "./dist/vue2/umd/index.js",
```

Comment:
we should track the vue 3 bundles too

Reply:
added! https://github.com/algolia/vue-instantsearch/pull/952/commits/7be3418e89e5cea0a9beabecb73588cbb0988034